src/kOS/Binding/FlightStats.cs

```diff
@@ -12,15 +12,17 @@ public class FlightStats : Binding
     {
         public override void AddTo(SharedObjects shared)
         {
-            shared.BindingMgr.AddGetter("ALT_APOAPSIS", () => shared.Vessel.orbit.ApA);
-            shared.BindingMgr.AddGetter("ALT_PERIAPSIS", () => shared.Vessel.orbit.PeA);
-            shared.BindingMgr.AddGetter("ALT_RADAR", () => Convert.ToDouble(shared.Vessel.heightFromTerrain > 0 ? Mathf.Min(shared.Vessel.heightFromTerrain, (float)shared.Vessel.altitude) : (float)shared.Vessel.altitude));
+            shared.BindingMgr.AddGetter("ALT", () => new Alt(shared));
+            shared.BindingMgr.AddGetter("ALT_APOAPSIS", () => (new Alt(shared)).GetApoapais());
```

Comment:
`ALT:APOAPSIS` still works, and `ALT_APOAPSIS` was, to my knowledge, never part of the documented binding list. we can leave it if the _ version is in wide use.